asmcli/commands/experimental/install.sh

```diff
@@ -15,7 +15,8 @@ x_install() {
   local CLUSTER_LOCATION; CLUSTER_LOCATION="$(context_get-option "CLUSTER_LOCATION")"
   local CLUSTER_NAME; CLUSTER_NAME="$(context_get-option "CLUSTER_NAME")"
 
-  # TODO: MANAGED_CNI install should go here, if necessary
+  install_mananged_cni
```

Comment:
Let's add another TODO here to remove this when AFC based install is ready.

Reply:
Done